src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,17 +1,27 @@
 package edu.kis.vh.nursery;
 
-public class HanoiRhymer extends defaultCountingOutRhymer {
+import edu.kis.vh.nursery.base_data_types.IntStack;
+
+public class HanoiRhymer extends DefaultCountingOutRhymer {
+
+	int totalRejected = 0;
+
+    public HanoiRhymer(IntStack stack) {
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.